visualization/app/codeCharta/state/store/fileSettings/markedPackages/markedPackages.service.spec.ts

```diff
@@ -66,4 +66,6 @@ describe("MarkedPackagesService", () => {
 			expect($rootScope.$broadcast).not.toHaveBeenCalled()
 		})
 	})
+
+	describe("onFilesChanged", () => {})
```

Comment:
Do we need this ?

Reply:
No, I wanted to add a test for this method, but the merging methods are tested in their own file already. Perhaps we can remove the merger tests and add them to this method here.

Removed this block for now though. :)